groups/prometheus/module-prometheus/cloud-init/templates/prometheus.yml.tpl

```diff
@@ -17,6 +17,14 @@ write_files:
           scrape_timeout: 15s
           metrics_path: /metrics
           scheme: http
-          static_configs:
-          - targets:
-            - ${prometheus_web_fqdn}:${prometheus_metrics_port}
+          ec2_sd_configs:
+            - region: eu-west-2
+              port: ${prometheus_metrics_port}
+          relabel_configs:
+              # Only monitor Concourse instances
```

Comment:
This comment about Concourse should come out as this is meant to be non-specific being a template